opentelemetry-tools-google-cloud/tests/test_cloud_trace_propagator.py

```diff
@@ -29,9 +29,10 @@
     get_hexadecimal_trace_id,
 )
 
+# def get_dict_value(dict_object: typing.Dict[str, str], key: str) -> str:
```

Comment:
Is this meant to be temporary?
If it's no longer needed better to delete completely.

Reply:
Oops yup